.github/workflows/pull_request.yml

```diff
@@ -157,7 +157,7 @@ jobs:
 
       - name: Build
         working-directory: arrow-libs
-        run: ./gradlew watchosX64Test watchosX86Test watchosSimulatorArm64Test watchosArm32TestBinaries watchosArm32TestKlibrary tvosX64Test tvosSimulatorArm64Test tvosArm64TestBinaries tvosArm64TestKlibrary
+        run: ./gradlew allTests
```

Comment:
Now we're running all Apple tasks twice, no? 
Before we tried splitting all targets across different actions to achieve the shortest total CI time as possible.

Reply:
Yeah the CI will be larger because it will run jvm tests and so on, I can set the concrete Gradle tests but it is easy to forget to add new targets in the future, what do you think?

Is there a lot of difference?

Reply:
Yes, that's a good point. We can try to use `allTests`, but then we should remove one of the two macos jobs. Let's see how long it takes to run all Apple targets on a single machine.